scripts/migrate-embeddings.ts

```
@@ -0,0 +1,412 @@
+    switch (action) {
+      case 'setup':
+        await migrator.createMigrationTable();
+        await migrator.createParallelSearchFunction();
+        break;
+        
+      case 'migrate':
+        const limit = args.includes('--limit') 
+          ? parseInt(args[args.indexOf('--limit') + 1])
+          : undefined;
+        
+        const category = args.includes('--category')
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)